packages/react/src/create-strict-api/__tests__/__fixtures__/strict-api.ts

```diff
@@ -0,0 +1,12 @@
+import { createStrictAPI } from '../..';
+
+const { css, XCSSProp, cssMap, cx } = createStrictAPI<{
+  '&:hover': {
+    color: 'var(--ds-text)';
+    background: 'var(--ds-surface-hover)' | 'var(--ds-surface-sunken-hover)';
+  };
+  background: 'var(--ds-surface)' | 'var(--ds-surface-sunken';
+  bkgrnd: 'red' | 'green';
```

Comment:
Currently passing unknown values isn't a type violation. I want to spend some time to see if it could be.

Reply:
I looked at this: The excess property check is killing us. Going to leave it — if anyone has ideas LMK!